doc/operator-monitoring-resources.md

```diff
@@ -27,25 +25,114 @@ spec:
     enabled: true
 ```
 
+NOTE: All monitoring resources will be created by the operator using *Create only* reconcilliation policy. That means that PrometheusRules and GrafanaDashboards objects can be updated preventing the operator to revert the changes back. This policy allows to tune, for instance, the alert thresholds, to your needs.
+
 ## Monitored components
 
 * Kubernetes resources at namespace level where 3scale is installed
 * Apicast Staging
 * Apicast Production
 * 3scale Backend worker
 * 3scale Backend listener
-* System sidekiq
+* System
 * Zync
 * Zync-que
 
-## Exposing monitoring resources
+## Monitoring stack
+
+3scale monitoring is leveraged by [prometheus](https://prometheus.io/) and [grafana](https://grafana.com/) monitoring solutions. They need to be up and running in the cluster and configured to watch for monitoring resources.
+
+Monitoring stack setup and running is beyond 3scale operator reponsabilities.
+There are many ways to provide the required monitoring stack. Few examples are given:
+
+* Openshift new proposal [User Workload Monitoring](https://github.com/openshift/enhancements/blob/master/enhancements/monitoring/user-workload-monitoring.md)
+
+* [kube-prometheus](https://github.com/coreos/kube-prometheus): This repository collects Kubernetes manifests, Grafana dashboards, and Prometheus rules combined with documentation and scripts to provide easy to operate end-to-end Kubernetes cluster monitoring with Prometheus using the Prometheus Operator.
+
+* For devtesting purposes only, *quickstart steps* to deploy minimum required monitoring stack in [monitoring-stack-deployment](monitoring-stack-deployment/README.md)
+
+### Prometheus
 
-`GrafanaDashboard` resources will all be labeled with
+3scale monitoring requires [Prometheus Operator](https://github.com/coreos/prometheus-operator) to be deployed in the cluster.
+The prometheus operator is an operator that creates, configures, and manages Prometheus clusters atop Kubernetes. It provides `PodMonitor`, `ServiceMonitor` and `PrometheusRule` custom resources definitions required by 3scale monitoring.
 
+Tested releases:
+* Prometheus operator `v0.32.0`
+* Prometheus image: `quay.io/openshift/origin-prometheus: 4.2`
+
+Make sure prometheus services are configured to monitor 3scale monitoring resources.
+The simplest configuration is catch-all config in `Prometheus` custom resource spec :
```

Comment:
The simplest way is using namespace selector like the ones we use in our dev clusters :
```
  podMonitorNamespaceSelector:
    matchExpressions:
      - key: monitoring-key
        operator: In
        values:
          - middleware

  serviceMonitorNamespaceSelector:
    matchExpressions:
      - key: monitoring-key
        operator: In
        values:
          - middleware
```

Reply:
Added some info about installing prometheus in different namespace than 3scale